python/src/equistore/operations/multiply.py

```diff
@@ -9,14 +9,17 @@
 
 
 def multiply(A: TensorMap, B: Union[float, TensorMap]) -> TensorMap:
-    """Return a new :class:`TensorMap` with the values being the element-wise
+    r"""Return a new :class:`TensorMap` with the values being the element-wise
     multiplication of ``A`` and ``B``.
 
     If ``B`` is a :py:class:`TensorMap` it has to have the same metadata as ``A``.
 
-    If gradients are present in ``A`` a multiplication is only performed if ``B`` is
-    a :py:class:`TensorMap` as well. The multiplication is performed with the
-    rule of the derivatives.
+    If gradients are present in ``A`` a multiplication (:math:`*`) is only performed if
+    ``B`` is a :py:class:`TensorMap` as well. The multiplication is performed with the
+    rule of the derivatives:
+
+    .. math::
+        \nabla(A * B) = B * \nabla A + A * \nabla B
 
     :param A: First :py:class:`TensorMap` for the addition.
     :param B: Second instance for the multiplication. Parameter can be a scalar a array
```

Comment:
Here it says that "Parameter can be a scalar **a array**..." - can ``B`` be an array? It looks like there are only methods for scalars (``_multiply_block_constant``) and blocks with the same dimensions (``_multiply_block_block``). 

If not, just edit so it reads "Parameter can be a scalar or a :py:class"`TensorMap`...". Apart from that everything looks good :)

Reply:
Ohh yes. I think this is a leftover from our draft.